modules/samples/producers/http/build.xml

```diff
@@ -0,0 +1,77 @@
+<!--
+~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~
+~ WSO2 Inc. licenses this file to you under the Apache License,
+~ Version 2.0 (the "License"); you may not use this file except
+~ in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~    http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing,
+~ software distributed under the License is distributed on an
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+
+<project default="run">
+
+    <property name="carbon.home" value="../../.."/>
+    <property name="src.dir" value="src/main/java"/>
+    <property name="resources.dir" value="src/main/resources"/>
+    <property name="temp.dir" value="temp"/>
+    <property name="lib.dir" value="../../lib"/>
+    <property name="class.dir" value="${temp.dir}/classes"/>
+    <property name="main-class" value="org.wso2.carbon.sample.http.Http"/>
+    <property name="username" value="admin"/>
+    <property name="password" value="admin"/>
+
+    <property name="url" value=""/>
+    <property name="sn" value=""/>
+    <property name="filePath" value=""/>
+
+    <path id="javac.classpath">
+        <pathelement path="${class.dir}"/>
+        <fileset dir="${lib.dir}"/>
+        <fileset dir="${carbon.home}/repository/components/plugins/"/>
```

Comment:
It is not correct to load all the jar files which are in plugins directory to class path. This will cause issues when running in windows environment.. We need to fix this..